nemo_skills/inference/model/tool_call.py

```
@@ -88,17 +84,21 @@ async def _execute_tool_call(self, tool_call, request_id: str):
-    async def _execute_tool_calls(self, tool_calls: List, request_id: str):
-        tasks = [self._execute_tool_call(tool_call, request_id=request_id) for tool_call in tool_calls]
+    async def _execute_tool_calls(self, tool_calls: List, request_id: str, completion_type: CompletionType):
+        tasks = [
+            self._execute_tool_call(tool_call, request_id=request_id, completion_type=completion_type)
+            for tool_call in tool_calls
+        ]
         tool_results = await asyncio.gather(*tasks)
         return [
-            self.response_formatter.format(tool_call, tool_result)
+            format_tool_response_by_completion_type(tool_call, tool_result, completion_type)
             for tool_call, tool_result in zip(tool_calls, tool_results)
```

Ruff (0.13.3)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)